tests/phpunit/CRM/Case/BAO/CaseTest.php

```diff
@@ -136,6 +136,113 @@ private function assertCasesOfUser($loggedInUser, $caseId, $caseCount) {
     $this->assertEquals($caseCount, count($caseRoles), 'Total case roles for logged in users must be ' . $caseCount);
   }
 
+  /**
+   * core/issue-1623: My Case dashlet doesn't sort by name but contact_id instead
```

Comment:
Do you use an IDE -  phpstorm highlights the need to declare the exception here -    * @throws \CRM_Core_Exception